.github/workflows/ios.yml

```diff
@@ -52,6 +52,13 @@ jobs:
           GIT_TAG: ${{ steps.vars.outputs.tag }}
         run: sed -i '' "s/COMMIT/$GITHUB_SHA/" debugInfo.json; sed -i '' "s/TAG/$GIT_TAG/" debugInfo.json
 
+      - name: Hardcode OAuth prefix and urls for production version
+        if: startsWith(github.ref, 'refs/tags/') && !endsWith(github.ref, '-beta')
+        run: |
+          sed -i '' "s/string>devmapswipe/string>mapswipe/" ios/mapswipe/Info.plist
+          sed -d '' "s/dev-mapswipe.web.app/mapswipe.web.app/" debugInfo.json
```

Comment:
I'm not sure if `mapswipe.web.app` is the right url for production. It could also be `msf-mapswipe`. However, the url is already in use by the leaderboard... https://msf-mapswipe.web.app/

Reply:
Good question. I didn't think about it. It looks like we could add `mapswipe.web.app`, or we could do `auth.mapswipe.org` which would look more legit for users who pay attention.

Reply:
Let's use `dev-auth.mapswipe.org` and `auth.mapswipe.org` for the dev and production apps respectively.